src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -158,15 +158,21 @@
     @Deprecated
     public transient String slaveCommandSuffix;
 
+    @Deprecated
+    public transient boolean usePrivateDnsName;
```

Comment:
May I ask why this was declared transient?

Reply:
Because it was deprecated?

Reply:
See https://github.com/jenkinsci/ec2-plugin/pull/400